external-api/src/http/admin.rs

```diff
@@ -35,3 +37,10 @@ pub struct OpenOrdersResponse {
     /// The open orders
     pub orders: Vec<OrderMetadata>,
 }
+
+/// The response to an "order metadata" request
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct OrderMetadataResponse {
```

Comment:
nit: `AdminOrderMetadataResponse`; this makes it more clear where the type will be used